### PR TITLE
Add release notes for 7.9.3

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: docs/reference/release-notes/7.9.asciidoc [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+
+SQL::
+* Fix exception when using CAST on inexact field {es-pull}62943[#62943] (issue: {es-issue}60178[#60178])
+
+Search::
+* Async search should retry updates on version conflict {es-pull}63652[#63652] (issue: {es-issue}63213[#63213])
+
+Transform::
+* Fix possible NPE if transform task has no node assigned {es-pull}62946[#62946] (issue: {es-issue}62847[#62847])
+* [ML][Transform] Filter null objects from field caps request {es-pull}62945[#62945]
```
Comment (jrodewig) [inline docs/reference/release-notes/7.9.asciidoc:56]:
```suggestion
* Filter null objects from field caps request {es-pull}62945[#62945]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 2]
```diff
@@ -1,3 +1,70 @@
...
+Features/Data streams::
+* Fix querying a data stream name in _index field. {es-pull}63170[#63170]
+
+Features/Ingest::
+* Handle error conditions when simulating ingest pipelines with verbosity enabled {es-pull}63327[#63327] (issue: {es-issue}63199[#63199])
+* Make for each processor resistant to field modification {es-pull}62791[#62791] (issue: {es-issue}62790[#62790])
+
+Machine Learning::
+* Fix online updates with custom rules referencing filters {es-pull}63057[#63057] (issue: {es-issue}62948[#62948])
+* Reset reindexing progress when DFA job resumes with incomplete r… {es-pull}62772[#62772]
```
Comment (jrodewig) [inline docs/reference/release-notes/7.9.asciidoc:46]:
```suggestion
* Reset reindexing progress when DFA job resumes with incomplete reindexing {es-pull}62772[#62772]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 3]
```diff
@@ -1,3 +1,70 @@
...
+
+EQL::
+* Avoid filtering on tiebreakers {es-pull}63215[#63215] (issue: {es-issue}62781[#62781])
+* Fix bug in sequences with any pattern {es-pull}63007[#63007] (issue: {es-issue}62967[#62967])
+
+Engine::
+* Fix to actually throttle indexing on getting activated {es-pull}61768[#61768]
+
+Features/Data streams::
+* Fix querying a data stream name in _index field. {es-pull}63170[#63170]
```
Comment (jrodewig) [inline docs/reference/release-notes/7.9.asciidoc:38]:
```suggestion
* Fix querying a data stream name in index field {es-pull}63170[#63170]
```

File: docs/reference/release-notes/7.9.asciidoc [hunk 4]
```diff
@@ -1,3 +1,70 @@
...
+* [ML][Transform] Filter null objects from field caps request {es-pull}62945[#62945]
+
+
+
+[[upgrade-7.9.3]]
+[float]
+=== Upgrades
+
+Infra/Packaging::
+* Switch bundled jdk back to Oracle JDK {es-pull}63288[#63288] (issue: {es-issue}62709[#62709])
```
Comment (jrodewig) [inline docs/reference/release-notes/7.9.asciidoc:65]:
```suggestion
* Switch bundled JDK back to Oracle JDK {es-pull}63288[#63288] (issue: {es-issue}62709[#62709])
```

**Review by jrodewig — APPROVED** [review]
LGTM. I left some minor suggestions to remove unneeded labels, fix truncation, remove punctuation, and other minor nits.

I'm also going to add @lcawl as a reviewer for the ML section.

Thanks @matriv!

File: docs/reference/release-notes/7.9.asciidoc [hunk 5]
```diff
@@ -1,3 +1,70 @@
...
+Features/Data streams::
+* Fix querying a data stream name in _index field. {es-pull}63170[#63170]
+
+Features/Ingest::
+* Handle error conditions when simulating ingest pipelines with verbosity enabled {es-pull}63327[#63327] (issue: {es-issue}63199[#63199])
+* Make for each processor resistant to field modification {es-pull}62791[#62791] (issue: {es-issue}62790[#62790])
+
+Machine Learning::
+* Fix online updates with custom rules referencing filters {es-pull}63057[#63057] (issue: {es-issue}62948[#62948])
+* Reset reindexing progress when DFA job resumes with incomplete reindexing {es-pull}62772[#62772]
```
Comment (lcawl) [inline docs/reference/release-notes/7.9.asciidoc:46]:
```suggestion
* Reset reindexing progress when data frame analytics job resumes with incomplete reindexing {es-pull}62772[#62772]
```

**Review by lcawl — APPROVED** [review]
ML content LGTM, thanks!

File: docs/reference/release-notes/highlights.asciidoc [hunk 6]
```diff
@@ -37,6 +37,27 @@ Other versions:
 //
 // Description.
 
+// tag::notable-highlights[]
+[discrete]
+=== Cross-cluster replication
+
+Fixed retries for {ccr}.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:44]:
```suggestion
=== Fixed retries for {ccr}
```

**Comment by matriv** [comment]
@jrodewig Could you please take a look at the release highlights which were just added?

File: docs/reference/release-notes/highlights.asciidoc [hunk 7]
```diff
@@ -37,6 +37,27 @@ Other versions:
...
+
+{ccr-cap} now retries operations that failed due to a circuit breaker or a lost
+remote cluster connection.
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+=== Engine
+
+Fix to actually activate throttle indexing.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:53]:
```suggestion
=== Fixed index throttling
```

File: docs/reference/release-notes/highlights.asciidoc [hunk 8]
```diff
@@ -37,6 +37,27 @@ Other versions:
...
+
+When indexing data into {es}, heap memory is used in lucene and {es} for buffering data. To keep the memory usage under
+control, {es} will move data to disk according to the <<indexing-buffer, indexing buffer settings>>. If moving
+data cannot keep up with the ongoing indexing, {es} will start throttling indexing. This throttling behavior was broken,
+so throttling was not activated, and it has been fixed.
+// end::notable-highlights[]
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights.asciidoc:59]:
```suggestion
When indexing data, {es} and Lucene use heap memory for buffering. To control
memory usage, {es} moves data from the buffer to disk based on your
{ref}/indexing-buffer.html[indexing buffer settings]. If ongoing indexing
outpaces the relocation of data to disk, {es} will now throttle indexing. In
previous {es} versions, this feature was broken and throttling was not
activated.
// end::notable-highlights[]
```

**Review by jrodewig — APPROVED** [review]
This LGTM. I left some suggestions, but you can ignore them if wanted.

However, you will need to change `<<indexing-buffer, indexing buffer settings>>` to
`{ref}/indexing-buffer.html[indexing buffer settings]` regardless. Thanks @matriv!

**Comment by jrodewig** [comment]
Gonna merge this in so it's ready for the release.

**Pull request merged**